unit/size_left_test.c

```diff
@@ -142,11 +142,10 @@ rx_size_left_err(void)
 	}
 
 	/* ep starts in a non-enabled state, may fail, should not SEGV */
-	fi_rx_size_left(ep);
+	(void) fi_rx_size_left(ep);
```

Comment:
Why the cast to void? Is it to silence a warning about the return value? Maybe we should be checking the return value